ajax/libs/nivo-lightbox/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "nivo-lightbox",
+  "filename": "nivo-lightbox.min.js",
+  "version": "1.2.0",
+  "description": "A simple, flexible, responsive, retina-ready jQuery lightbox plugin",
+  "homepage": "https://dev7studios.com/nivo-lightbox",
+  "keywords": [
+    "image",
+    "lightbox",
+    "javascript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:gilbitron/Nivo-Lightbox.git"
+  },
+  "license": "MIT",
+  "author": "Gilbert Pellegrom <gilbert@pellegrom.me>",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/gilbitron/Nivo-Lightbox.git",
+    "basePath": "",
+    "files": [
+      "nivo-lightbox*",
+      "themes/default/*"
```

Comment:
static `"default"` is not flexible at all, try: `"themes/**/*.+(css|js|map|png|gif|jpg)"`